test/functional/Java21andUp/build.xml

```diff
@@ -77,7 +77,12 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<target name="build">
 		<if>
 			<not>
-				<matches string="${JDK_VERSION}" pattern="^(8|9|1[0-9]|20)$$" />
+				<!--
+				The test suites are temporarily disabled in JDK21 as
+				the related code has yet been updated to support the latest
```

Comment:
I think the word "not" is missing in "has not yet been".

Reply:
Fixed.